ConsoleGUI/Controls/ListView.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConsoleGUI.Data;
+using ConsoleGUI.Input;
+using ConsoleGUI.UserDefined;
+
+namespace ConsoleGUI.Controls
+{
+    public class ListView : SimpleControl, IInputListener
+    {
+        public Color TextColor { get; set; } = Color.White;
```

Comment:
Both the `TextColor` and the `SelectedTextColor` should be using the `Setter` helper class and backing fields (see other controls).
With the current implementation, changing those colors after the control has been rendered will not trigger an update.

Reply:
I will implement setters but be aware that this control implements SimpleControl and so the Initialize functions that other controls seem to have aren't available.